Sources_v3/StreamChatUI/Utils/UIView+UIConfig.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - Backing type-erased config
+
+private extension UIView {
+    static var anyUIConfigKey: UInt8 = 0
+    
+    var anyUIConfig: Any? {
+        get { objc_getAssociatedObject(self, &Self.anyUIConfigKey) as? String }
+        set { objc_setAssociatedObject(self, &Self.anyUIConfigKey, newValue, .OBJC_ASSOCIATION_RETAIN) }
+    }
+}
+
+// MARK: - Public config API
+
+public extension UIView {
+    func register<T: UIExtraDataTypes>(config: UIConfig<T>) {
+        anyUIConfig = config
+    }
```

Comment:
That's how we can override the `UIConfig`. The new config will be applied for all the views down the hierarchy.